src/main/java/com/service/bank/repository/RoleRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.service.bank.repository;
+
+import com.service.bank.model.Role;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+
+    @Query("from Role r where r.role = :roleName")
+    Optional<Role> getRoleByName(String roleName);
```

Comment:
```suggestion
    Optional<Role> getByRoleName(RoleName name);
```
Spring Data JPA is capable of automatically build the query in many cases based on the method name and parameters)